app/views/index/tos.phtml

```diff
@@ -27,3 +27,9 @@
 	<h1><?= _t('index.tos.title')?></h1>
 	<?= $this->terms_of_service ?>
 </main>
+
+<?php if (!FreshRSS_Auth::hasAccess()) { ?>
```

Comment:
Probably fine, but why the special case here?

Reply:
If the user is not logged in then he/she has no navigation to switch to the about page from the TOS.

If the user is logged in, then he/she will find the TOS in the configs section, that has a navigation menu on the left hand side, where the About page is linked to. In the logged-in state the page does not have a footer.